src/Faithlife.Build/BuildApp.cs

```diff
@@ -66,16 +66,31 @@ public BuildTarget Target(string name)
 		return target;
 	}
 
+	/// <summary>
+	/// Adds an action to execute when the command line arguments have been parsed, but before the target(s) are evaluated and executed.
+	/// </summary>
+	/// <param name="commandLineParsed">The action to execute when the command line arguments have been parsed.</param>
+	public void CommandLineParsed(Action commandLineParsed)
```

Comment:
```suggestion
	public void CommandLineParsed(Action action)
```
`commandLineParsed` seems redundant with the method name to me, but I don't feel strongly. Merge at will!